### ISSUE TITLE
Adding "rename" button for files and folders

### ISSUE THREAD

**Issue by mtaghiza** [idies/scidrive-ui#36]
Can't change names of files or folder; could add a "rename" button.

**Comment by dimm0** [comment]
There's no rename operation in Swift. The only way to rename is to copy with a new name and remove the source. Implementing rename this way can cause a user to "rename" a large folder and trigger a huge undesired operation of moving files.

**Comment by jkim485** [comment]
I think that This should be explained in help page.

**Comment by jkim485** [comment]
I agree with your comment that this must be made clear to the users.

I also think that this is a major negative feature.  I can’t think of an single system that does not allow file/object renaming – how the thing is internally stored should have nothing to do with how it is displayed.

What are your thoughts on this?

Mike

**Comment by jkim485** [comment]
I think that renaming file/folder must be supported as Mike described. however this is implemented under the cover, it has to be done. 
If it is just difficult to implement it right away, as for now at least users should be warned properly in the help so users do not get confused.
